library/connect-web/v0.2.1/buf.plugin.yaml

```diff
@@ -0,0 +1,16 @@
+version: v1
+name: buf.build/library/connect-web
+plugin_version: v0.2.1
+source_url: https://github.com/bufbuild/connect-web
+description: Simple, reliable, interoperable. A better gRPC-web.
+deps:
+  - plugin: buf.build/library/protobuf-es:v0.1.1
```

Comment:
Updated this based on the peer dependencies here:

https://github.com/bufbuild/connect-web/blob/main/packages/protoc-gen-connect-web/package.json#L27